doc/user/zebra.rst

```diff
@@ -1058,6 +1058,13 @@ zebra Terminal Mode Commands
    nexthop groups that do have an afi. [type] allows you to filter those
    only coming from a specific NHG type (protocol).
 
+.. index:: show <ip|ipv6> zebra route dump [<vrf> VRFNAME]
+.. clicmd:: show <ip|ipv6> zebra route dump [<vrf> VRFNAME]
+
+   It dumps all the routes from RIB with detailed information including
+   internal flags,status ..etc. This is defined as hidden command and
```

Comment:
Please fix the basic English issues here

"internal flags, status etc."
"This is defined as a hidden command"

Reply:
Addressed .